CHANGELOG.md

```diff
@@ -15,9 +15,12 @@ be found.
 + Add support for comments in `.dockerignore` [#23111](https://github.com/docker/docker/pull/23111)
 * Support for UTF-8 in Dockerfiles [#23372](https://github.com/docker/docker/pull/23372)
 * Skip UTF-8 BOM bytes from `Dockerfile` and `.dockerignore` if exist [#23234](https://github.com/docker/docker/pull/23234)
+* windows: support for `ARG` to match Linux [#22508](https://github.com/docker/docker/pull/22508)
```

Comment:
Should we have the same style as the other entries? e.g.: "Support for `ARG` is now also available in Windows`?